src/app/pages/datasets/components/dataset-management/dataset-management.component.scss

```diff
@@ -318,3 +311,8 @@ $padding-left: 8px;
     }
   }
 }
+
+ix-tree {
+  max-height: calc(100vh - 330px);
```

Comment:
What stands for 330px / How is it gained?

Reply:
I just calculated it visually, I think it will be good to refactor datasets blocks, to have same height via flex somehow.

Reply:
> What stands for 330px / How is it gained?
I just calculated it visually, I think it will be good to refactor datasets blocks, to have same height via flex somehow.

Reply:
if we like that approach I found, I'll work on refactoring